src/window/concept_frame.rs

```diff
@@ -204,7 +204,7 @@ struct PointerUserData {
  */
 
 struct Inner {
-    parts: Vec<Part>,
+    parts: Option<Vec<Part>>,
```

Comment:
Maybe just keep the `Vec<_>` and test if it is empty to decide if the decorations have been initialized, rather than introducing an `Option` ?

Reply:
I don't really mind, I've picked option since it's easier to work with and is more clear when showing that value isn't there.

Reply:
true

Reply:
Updated.